### PR TITLE
HTML-CSS Assignment

### PR DESCRIPTION
**Pull request by vandanainnogenio**
Web capture_24-11-2022_92238_127 0 0 1
1. should use the flex property 
2. should use max-width property.
3. should specify width in %.
4. use rem and em.
5. use position property in the navigation bar.
6. avoid horizontal scroll bar.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+    font-size: 1.5rem;
+    margin-top: 6rem;
+      
+ }
+ .destext{
+    background-color: transparent;
+    border-radius: 20px;
+    width: 50%;
+    resize: none;
+    height: 15vh;
```
Comment (dev-innogenio) [inline styles.css:243]: use min-height

File: index.html [hunk 2]
```diff
@@ -1,12 +1,138 @@
...
+                </div>
+                <div class="signin">
+                    <a href="#" target="_blank"><i>SIGN IN /SIGN UP</i></a>
+                </div>
+            </div>
+            <div class="container">
+                 <div class="matter">
+                 <div class="welcome">Welcome to the competition
+                </div>
+                <span class="sentence">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Eget ac, nec viverra
```
Comment (dev-innogenio) [inline index.html]: use div instead of span

File: index.html [hunk 3]
```diff
@@ -1,12 +1,138 @@
...
+                <div class="click">
+                    <a href="#" target="_blank" >Click to see more</a>
+                </div>
+            </div>
+            <div class="plant">
+                <img src="asset/capture.PNG" alt="plant">
+            </div>
+            </div>
+        </div>
+        <div class="secondhalf">
```
Comment (dev-innogenio) [inline index.html]: use meaningful class names

File: index.html [hunk 4]
```diff
@@ -1,12 +1,138 @@
...
+            <div class="front">
+            <div class="leftside">
+            <div class="fill">Fill the Application</div>
+            <div class="form">
+                <form>
+                <div class="fullname">
```
Comment (dev-innogenio) [inline index.html]: use "-" between class names

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,333 @@
...
+}
+.awards{
+    font-size: 5rem;
+    font-weight:lighter;
+    padding: 2rem;
+}
+.items{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
```
Comment (dev-innogenio) [inline styles.css:43]: use space between or gap

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,333 @@
...
+    margin-left: 0px;
+    margin-top: 4rem;
+    text-decoration: underline;
+}
+.secondhalf{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: 15rem;
+    padding-top: 5rem;
```
Comment (dev-innogenio) [inline styles.css]: use padding inside one property

File: styles.css (hunk 1 above)
Comment (vandanainnogenio) [reply styles.css:243]: using min-height property, the line which is separating the second form (fill the application) is not touching its two ends.

**Pull request merged**